pyproject.toml

```diff
@@ -28,7 +28,7 @@ dependencies = [
     "matplotlib>=3.3,<3.9",  # Should make this optional with a "viz" extras
     "networkx>=2.2,<3.2",
     "numpy>=1.18.5,!=1.23.0,<1.27",
-    "pandas[parquet,excel,fss,gcp,compression]>=2.0,<2.1",
+    "pandas[compression,excel,fss,gcp,parquet,performance]>=2,<2.2",
```

Comment:
In addition to allowing pandas 2.1.x, I alphabetized these and included `performance` since it did result in a noticeable speedup in my testing in the `ferc-xbrl-extractor` repo (even though that didn't end up being the fix I used).